include/rocksdb/file_system.h

```diff
@@ -47,6 +47,21 @@ class RateLimiter;
 using AccessPattern = RandomAccessFile::AccessPattern;
 using FileAttributes = Env::FileAttributes;
 
+// File types in current RocksDB
+enum FileType : unsigned char {
```

Comment:
Why put this in file_system.h if it's used on options.h?

Reply:
Will move it to options.h.